views/components/table_card.erb

```diff
@@ -66,7 +66,20 @@
                   %>
                   class="<%= cls %> <%= col_opts[:extra_class] %>"
                 >
-                  <%== part("components/value_content", content: value, opts: col_opts) %>
+                  <% if value.is_a?(Array) %>
```

Comment:
`col_opts` is no longer considered for the `value_content` render if `value` is an array.  Maybe that isn't a problem, though?

Reply:
Since we pass `*_opts` for the first and second rows separately, I couldn't find a way to pass `col_opts`. We can ignore it for now.